sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogEventSuite.scala

```diff
@@ -104,6 +109,8 @@ class ExternalCatalogEventSuite extends SparkFunSuite {
       tableType = CatalogTableType.MANAGED,
       storage = storage,
       schema = new StructType().add("id", "long"))
+    val tableDefWithSparkVersion =
```

Comment:
Where do we need this val?

Reply:
Sorry this was from my original code, will update it.